android/forms-screenlet-demo/app/src/main/AndroidManifest.xml

```diff
@@ -57,6 +57,10 @@
 		</activity>
 		<activity android:name="com.liferay.mobile.formsscreenletdemo.view.WorkflowActivity">
 		</activity>
+		<activity android:name="com.liferay.mobile.formsscreenletdemo.view.MyPoliciesActivity">
```

Comment:
Define a better name for MyPoliciesActivity and PolicyActivity. Maybe PoliciesList and PolicyDisplay or something similar.

Reply:
Okak, I'll do it.